Error: Cannot find module 'gulp-concat'

I hit this build error. It looks like `gulp-concat` was added to `index.js` but still remains in `devDependencies`. From this recent commit: https://github.com/scniro/gulp-clean-css/commit/3104238c01fb0ebc5b32c0d61cf13788f9339b4d#diff-168726dbe96b3ce427e7fedce31bb0bcR7

I can add `gulp-concat` to my project's `devDependencies` to get around this but others may hit the same issue. Perhaps `gulp-concat` should be a strict `dependency`.

```
Error: Cannot find module 'gulp-concat'
```

Thanks for the project, and keep up the good work :)

--- a/package.json
+++ b/package.json
@@ -30,6 +30,7 @@
   ],
   "dependencies": {
     "clean-css": "^3.4.21",
+    "gulp-concat": "^2.6.1",
     "gulp-util": "^3.0.7",
     "object-assign": "^4.1.0",
     "through2": "^2.0.3",
@@ -41,7 +42,6 @@
     "del": "^2.2.2",
     "express": "^4.14.0",
     "gulp": "^3.9.1",
-    "gulp-concat": "^2.6.1",
     "gulp-istanbul": "^1.1.1",
     "gulp-mocha": "^3.0.1",
     "gulp-rename": "^1.2.2",

